client/app/oui-less-override.less

```diff
@@ -68,3 +68,9 @@
         border-left-width: 1px;
     }
 }
+
+.oui-tile-h-100 {
```

Comment:
Maybe you could apply bootstrap `.h-100` directly on `oui-tile` in templates ?

Reply:
it won't work. This is a workaround for the moment as oui-tile does not stretch to occupy its parent full height. I have created a bug on ui kit team to track this. The actual fix will come in 2.17.0 release. O will remove this once oui kit fixes this issue.